launcher/internal/gpu/driverinstaller.go

```diff
@@ -0,0 +1,123 @@
+// Package gpu provides the devices information for cos-gpu-installer
+package gpu
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/cio"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/containerd/oci"
+	"github.com/google/go-tpm-tools/launcher/spec"
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+const (
+	installerContainerID = "tee-gpu-driver-installer-container"
+	installerSnapshotID  = "tee-gpu-driver-installer-snapshot"
+)
+
+// DriverInstaller contains information about the gpu driver installer settings
+type DriverInstaller struct {
+	cdClient   *containerd.Client
+	launchSpec spec.LaunchSpec
+	logger     *log.Logger
+}
+
+// NewDriverInstaller instanciates an object of driver installer
+func NewDriverInstaller(cdClient *containerd.Client, launchSpec spec.LaunchSpec, logger *log.Logger) *DriverInstaller {
+	return &DriverInstaller{
+		cdClient:   cdClient,
+		launchSpec: launchSpec,
+		logger:     logger,
+	}
+}
+
+// InstallGPUDrivers installs the GPU driver on host machine using the cos-gpu-installer container.
+// It uses containerd package to launch and manage the driver installer container.
+func (di *DriverInstaller) InstallGPUDrivers(ctx context.Context) error {
```

Comment:
Can we just run the command using `exec.Command` from https://pkg.go.dev/cos.googlesource.com/cos/tools.git@v0.0.0-20241008015903-8431fe581b1f/src/cmd/cos_gpu_installer#section-readme to install GPU driver?

Reply:
Done. Added comment with README link.